internal/monitors/expvar/expvar.go

```diff
@@ -56,6 +56,10 @@ type Monitor struct {
 func (m *Monitor) Configure(conf *Config) error {
 	m.logger = log.WithFields(log.Fields{"monitorType": monitorType})
 
+	if m.Output.HasEnabledMetricInGroup(groupEnhanced) {
```

Comment:
I don't think we should use enhanced as the group. There are probably better groups to assign the metrics to. Just off the top of my head it looks like maybe a memory group and a gc group. But regardless it seems what this is really saying is `m.Output.IsAnyCustomMetricEnabled()`.

Reply:
Yeah, the method to test for any non-default metric seems cleaner.